internal/utils/debuggingconfigurations.go

```diff
@@ -29,15 +29,18 @@ const (
 	defaultInstanceName = "default"
 )
 
-// GetDebuggingConfiguration gets default DebuggingConfiguration
+// GetDebuggingConfiguration gets default DebuggingConfiguration in the specified namespace and cluster
 func (a *k8sAccess) GetDebuggingConfiguration(
 	ctx context.Context,
+	namespace string,
+	clusterName string,
 ) (*libsveltosv1alpha1.DebuggingConfiguration, error) {
 
 	req := &libsveltosv1alpha1.DebuggingConfiguration{}
 
 	reqName := client.ObjectKey{
-		Name: defaultInstanceName,
+		Namespace: namespace,
```

Comment:
here we need to distinguish two scenarios. 

If namespace is empty (and so clusterName and clusterType) we can use a.client (which points to the management cluster),

otherwise we need to [clusterproxy.GetSveltosKubernetesClient](https://github.com/projectsveltos/libsveltos/blob/fd9879699f8d4e43b1412ffcca3e253525d70a6d/lib/clusterproxy/clusterproxy.go#L142) to get the client pointing to the managed cluster.

Reply:
Should we also pass clusterType to GetSveltosKubernetesClient?

Reply:
sorry my bad. This is the [method](https://github.com/projectsveltos/libsveltos/blob/fd9879699f8d4e43b1412ffcca3e253525d70a6d/lib/clusterproxy/cluster_utils.go#L203) to use and that requires clusterType.